wrapanapi/systems/lenovo.py

```diff
@@ -348,20 +348,28 @@ def set_server_led_blinking(self, server_name):
 
         return "LED state action has been sent, status:" + str(response.status_code)
 
-    def server_stats(self, *requested_stats, **kwargs):
-        # Get the requester which represents the class of this method's caller
-        requester = kwargs.get('requester')
-
+    def server_stats(self, requester, requested_stats, **kwargs):
```

Comment:
Another thing that really bothers me is why we even call it "requester"? I have to dig code everytime because I always forget this is in fact "server" - so, couldn't we just call it `server`? :) We could event accept string `server_name` instead as we only access `requester.name` everywhere anyway. Looking forward to your opinion.

Reply:
I agree, since in this context this will always be a PhysicalServer changing this input to server or even physical_server would be more clear.

Reply:
Thank you! I went with `physical_server`.